appserver/tests/payara-samples/samples/payara-expression-config-properties/pom.xml

```diff
@@ -105,4 +112,36 @@
             </plugin>
         </plugins>
     </build>
-</project>
\ No newline at end of file
+    <profiles>
+        <profile>
+            <id>auto-skip-test</id>
```

Comment:
I'm not fond of this.
A test that is disabled due to not having an environment variable set is never run.

Reply:
Repeatable tests should run without hacking computer environment. By default it only fails.

Reply:
I understand that in this case there was no other way possible - I plan to move this test to TC PR including jenkinsfile etc., so final state should be clean next week ;)

Reply:
After this PR will make it to master I will rebase my TC branch and transform this test to TC.